docs/shared/reference/nx-json.md

```diff
@@ -265,6 +265,37 @@ If you are using distributed task execution and disable caching for a given targ
 
 {% /callout %}
 
+### Executor/command options
+
+You can configure options specific to a target's executor. As an example, if your repo have projects using the `@nx/js:tsc` executor, you can provide some default options as follows:
+
+```json {% fileName="nx.json" %}
+{
+  "targetDefaults": {
+    "@nx/js:tsc": {
+      "generateExportsField": true
+    }
+  }
+}
+```
+
+You can also provide defaults for [inferred targets](/concepts/inferred-tasks) or targets running a command using the `nx:run-commands` executor. As an example, if your repo has projects with a `build` inferred target running the `vite build` command, you can provide some defaults options as follows:
```

Comment:
This is a pretty dangerous example.  Should we even show people this?

Reply:
That's a good point. It's true that it's dangerous for projects with multiple plugins inferring multiple `build` targets with different commands, but there are projects that use a single tool to build and where this wouldn't be a dangerous thing to do, and it will be as valid as setting target defaults for explicit targets/executors.

I think the danger this poses makes it more important to have an example but we need to add a caution callout to explain the possible consequences. Even if we don't show an example, users might still add a target default and wouldn't know about the possible consequences. So, having an explicit example and calling out the potentially bad consequences in some scenarios would actually be more helpful.

Wdyt?

Reply:
I amended the PR with the rephrased example and the caution callout. Let me know what you think.